NewsApp/app/src/main/java/com/thomasvondollen/newsapp/views/ArticleView.kt

```diff
@@ -0,0 +1,34 @@
+package com.thomasvondollen.newsapp.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import com.thomasvondollen.newsapp.Article
+import com.thomasvondollen.newsapp.databinding.ArticleViewBinding
+
+class ArticleView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0
+): LinearLayout(context, attrs, defStyleAttr, defStyleRes) {
+
+    private val binding = ArticleViewBinding.inflate(LayoutInflater.from(context), this)
+
+    fun setArticleTitle(article: Article) {
+        binding.titleTextView.text = article.title
+    }
+
+    fun setArticleAuthor(article: Article) {
+        binding.authorTextView.text = article.author
```

Comment:
Ah, so the reason why this wasn't working is because the `orientation` which is set in the layout is now invalid/unused because `<merge>` tags only allow for `android:layout_width` and `android:layout_height`.  You'll want to programmatically set the orientation here in code with:

```kotlin
    init {
        orientation = VERTICAL
    }
 ```
 
 Alternatively, you could use a `ConstraintLayout` as your overridden base class here (instead of LinearLayout) and add the constraints in the XML layout child elements.